src/capital-project/capital-project.service.ts

```diff
@@ -20,8 +20,15 @@ export class CapitalProjectService {
     private readonly capitalProjectRepository: CapitalProjectRepository,
   ) {}
 
-  async findMany({ limit = 20, offset = 0 }: FindCapitalProjectsQueryParams) {
+  async findMany({
+    limit = 20,
+    offset = 0,
+    ccd = "",
+    cd = "",
```

Comment:
Similar feedback to the [managing agency PR](https://github.com/NYCPlanning/ae-zoning-api/pull/405#discussion_r1928897633). In sort, using `null` as value to represent the lack of `district`. Then in the repository, specifically checking for `null` rather than a falsy (empty string) value.  

```suggestion
    ccd = null,
    cd = null,
```

Reply:
Done.